Gordon360/Controllers/RecIM/ActivitiesController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class ActivitiesController : GordonControllerBase
+    {
+        private readonly IActivityService _activityService;
+
+        public ActivitiesController(IActivityService activityService)
+        {
+            _activityService = activityService;
+        }
+
+        ///<summary>Gets a list of all Activities by parameter </summary>
+        ///<param name="active"> Optional active parameter </param>
+        ///<param name="time"> Optional time parameter </param>
+        /// <returns>
+        /// All Existing Activities 
+        /// </returns>
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<ActivityViewModel>> GetActivities([FromQuery] DateTime? time, bool active)
```

Comment:
I would still change the parameter name here. "Time" is overloaded and someone looking over your code would not be able to understand at first glance whether this is start time, end time, or something else.

Reply:
Yeah, we're still discussing exactly what types of use cases we have. It's leaning towards just having the "future" "past" "all" tag but we're going to confirm with the customer next week.